### PR TITLE
Pull request format examples

### PR DESCRIPTION
**Pull request by mike-sheppard**
**Summary**
Added a  PR format / example to code review guide, ironically can't use the format on this PR as it's pretty minor ;)

### CODE REVIEW

File: code-review/README.md [hunk 1]
```diff
@@ -43,3 +50,31 @@
   - Do unit tests actually test that the code is performing the intended functionality?
   - Are arrays checked for ‘out-of-bounds’ errors?
   - Could any test code be replaced with the use of an existing API?
+
+
+## Pull Request Format
+  - Prefix the branch & PR with the Jira ticket ID eg. `[TD-000] Brief summary of branch`
+  - Include Jira ticket number, summary description of the changes, testing notes & example screenshots for UI changes.
+
+#### Example PRs
```
Comment (minsikzzang) [inline code-review/README.md]: we need 3 `#` :) not 4 ?

Comment (mike-sheppard) [inline code-review/README.md]: cool all sorted :)

File: code-review/README.md [hunk 2]
```diff
@@ -43,3 +50,30 @@
...
+### Example PRs
+  - https://github.com/factorymedia/coresites/pull/287
+  - https://github.com/factorymedia/coresites/pull/286
+  - https://github.com/factorymedia/coresites/pull/282
+
+### Pull Request Template
+```md
+**Jira Ticket**
+https://factorymedia.atlassian.net/browse/TD-xxx
+
```
Comment (minsikzzang) [inline code-review/README.md:74]: can we also have PB link?

File: code-review/README.md [hunk 3]
```diff
@@ -43,3 +50,30 @@
...
+**Summary**
+Quick description of changes and expected behaviour.
+
+**Example**
+A few links or images showing before / after changes.
+
+**Testing / QA Notes**
+Brief notes to save your tester some time, and let them know / what to test for.
+
+```
```
Comment (minsikzzang) [inline code-review/README.md:88]: can we also add review note? i.e.)

```
** Review note
I want a review only for coding style / typo please (30% review)
I want a review for all (styles / typo / logic / etc) (90% review)
```

File: code-review/README.md (hunk 2 above)
Comment (mike-sheppard) [inline code-review/README.md:74]: yup, just added it

File: code-review/README.md (hunk 3 above)
Comment (mike-sheppard) [inline code-review/README.md:88]: cool cool, added that one in too

**Comment by minsikzzang** [comment]
:+1:

**Pull request merged**